src/Linter/PhpLinter.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace OomphInc\WASP\Linter;
+
+class PhpLinter implements LinterInterface {
+
+	public function lint($code) {
+		// open a process
+		$process = proc_open('php -l', [
```

Comment:
would this work on windows? 🤔 @kalilsn

Reply:
I think this would work fine except for the `/dev/null` portion. Stackoverflow tells me the Windows equivalent is just called 'nul'

Reply:
yeah, that could just be replaced with a pipe that we never read tbh